vault/resource_kmip_secret_backend.go

```diff
@@ -0,0 +1,218 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-provider-vault/util"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var kmipAPIFields = []string{
+	"default_tls_client_key_bits",
+	"default_tls_client_key_type",
+	"default_tls_client_ttl",
+	"listen_addrs",
+	"server_hostnames",
+	"server_ips",
+	"tls_ca_key_bits",
+	"tls_ca_key_type",
+	"tls_min_version",
+}
+
+func kmipSecretBackendResource() *schema.Resource {
+	return &schema.Resource{
+		Create: kmipSecretBackendCreate,
+		Read:   kmipSecretBackendRead,
+		Update: kmipSecretBackendUpdate,
+		Delete: kmipSecretBackendDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"path": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				Description:  "Full path where the generic secret will be written",
```

Comment:
Shouldn't that be talking about the backend's mount path?

Reply:
Yes, fixed in 8d85ef7